data/JobRunner/JobRunner.csproj

```diff
@@ -0,0 +1,36 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>net8.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
```

Comment:
Minor thing but the decision was taken awhile back that we wouldn't be using nullable reference types in MYA yet, so this should be disable atm. 
Personally I wanted to use them, but it'd be too big of a refactor so it's a future tech debt concern